### ISSUE TITLE
🐛 CanadaLogin phone number issue

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#17094]
### 🛡️Reported during Fable testing: 

**Task:**
Select your prefered verification method as sms, enter your phone number (provided in Environment), Return to Mailosaur account and copy the 6-didget code to continue, skip the recovery step and select continue.

**Error reported by tester:**
Even though everywhere else, the commands of Dragon are working really well, **when trying to type or paste the phone number itself, nothing seems to be happening in the text field**, so I am noting it as an issue here, just in case. I'm not sure why exactly only that part is not working, but I would recommend fixing it to make it more compatible with the commands, so it's easier overall to fill in the phone number itself.

**Expected behaviour:**
User should be able to paste or type numbers

### 🕵️ Details

**Operating system**: Windows
**Browser**: Chrome
**Assistive technology**: Dragon NaturallySpeaking

### 🔄 Regression

No

**Comment by mnigh** [comment]
@BrookeCameron is this behaviour happening on talent.canada.ca or on the CanadaLogin site? if it's not on talent.canada.ca, the issue should be raised with the CanadaLogin team as we don't have access to their code.
